doc/man7/provider-encoder.pod

```diff
@@ -178,6 +183,9 @@ hints, but must do so with care.  In the end, the output must make sense,
 and if there's a corresponding decoder, the resulting decoded object must
 match the original object that was encoded.
 
+OSSL_FUNC_encoder_does() should tell if a particular implementation supports
```

Comment:
Why not name it OSSL_FUNC_encoder_supports instead? I was confused when I first saw the name.

Reply:
I was thinking of the `has` keymgmt functions...  I'll ponder a bit on this.

Reply:
After a bit more deliberation on the subject in https://github.com/openssl/openssl/pull/13248#pullrequestreview-520422434, I decided on `does_selection`.  I hope that's satisfactory enough.